### PR TITLE
Topic - IgxExpansionPanel - Add Topic and demos for control

### LINKED ISSUE #668: Docs - IgxExpansionPanel
Add page(s) for IgxExpansionPanel

[Spec](https://github.com/IgniteUI/igniteui-angular/wiki/Expansion-Panel-Specification)
[Issue](https://github.com/IgniteUI/igniteui-angular/issues/307)

### PR DESCRIPTION
**Pull request by ViktorSlavov**
Closes #668

### CODE REVIEW

File: en/components/expansion_panel.md [hunk 1]
```diff
@@ -34,19 +34,21 @@ We can now start using `igx-expansion-panel` in the markup. The expansion panel
         - `igx-expansion-panel-description` - stores the component description. Can be used to provide a short summary. Will always be displayed second in the headers.
```
Comment (IvayloG) [inline en/components/expansion_panel.md]: "always appear at the beginning of the header." Could be misleading as the icon location is not considered.

Comment (IvayloG) [inline en/components/expansion_panel.md]: "Will always be displayed second in the headers" could be misleading as there could be no title at all. Can be something like after a title if title is set.

File: en/components/expansion_panel.md [hunk 2]
```diff
@@ -63,9 +65,217 @@ The `IgxExpansionPanelComponent` exposes the `.igx-expansion-panel` class which
...
+This way, the description will only be display when the component is collapsed.
+
+If we want to add more complex functionality depending on the component's state, we could also bind to the event emitters.
+```typescript
+// in expansion-panel.component.ts
+
+export class ExpansionPanelComponent {
+    ...
+    public handleExpansion(args: {event: Event}): void {
+        ... // Some functionality
```
Comment (IvayloG) [inline en/components/expansion_panel.md]: It will be best to give some visual feedback for the user what is the outcome of this eventHandler like "//Check console" or  else.

Comment (ViktorSlavov) [reply en/components/expansion_panel.md]: This is addressed in the samples PR

File: en/components/expansion_panel.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
+---
+title: Expansion Panel
+_description: Provides an easily configurable component with two states - collapsed and expanded.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Collapsible component, IgxExpansionPanel, Ignite UI Expansion Panel
+---
+
+## Expansion Panel
+<p class="highlight">
+The `IgxExpansionPanel` is a lightweigh component which can be rendered in two states - collapsed or expanded. When collapsed, the panel displays a short summary of its content. The collapsed panel can be toggled (via click or keyboard interactions) to display the whole body of the component, containing any additional necessary content.
```
Comment (bazal4o) [inline en/components/expansion_panel.md]: Let's reword second sentence to The collapsed panel can be toggled using mouse click, or keyboard interactions (maybe have link here) to display the body completely, containing....

File: en/components/expansion_panel.md [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="expansion-sample-2-sample" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+### Styling the header
+The `IgxExpansionPanelComponent` allows for easy customization of the header.
+The default icon for the toggle state of the control can be templated. The position of this icon is also configurable - either the start or end of the header. It can also be hidden.
+
+Configuring the position of the header icon can be done through the `iconPosition` input on the `igx-expansion-panel-header`. The posible options for the icon position are **left**, **right** and **none**. Let's configure our components button to go on the *right* side.
```
Comment (bazal4o) [inline en/components/expansion_panel.md]: Maybe reword the last sentence to - The next code sample demonstrates how to configure the component's button to go on the *right* side

File: en/components/expansion_panel.md [hunk 5]
```diff
@@ -0,0 +1,281 @@
...
+...
+    <igx-expansion-panel-header>
+        ...
+        <igx-expansion-panel-description *ngIf="panel.collapsed">
+            One of the smallest birds
+        </igx-expansion-panel-description>
+    </igx-expansion-panel-header>
+```
+
+This way, the description will only be display when the component is collapsed.
```
Comment (bazal4o) [inline en/components/expansion_panel.md]: Move this before the code snippet (html) -> Using the following code sample will show the description only when the component is in collapsed state.

File: en/components/expansion_panel.md [hunk 6]
```diff
@@ -0,0 +1,281 @@
...
+    ...
+})
+export class AppModule {}
+```
+
+We can now start using `igx-expansion-panel` in the markup. The expansion panel has a simple structure, as you can see below:
+- `igx-expansion-panel` - the component host - stores header and body
+    - `igx-expansion-panel-header` - stores the component header. This is always visible. Stores title and description as well as any additional content
+        - `igx-expansion-panel-title` - stores the component title. Has default styling. Will always appear first in the header content (after icon if `iconPosition === 'left'`). Used in aria labels.
+        - `igx-expansion-panel-description` - stores the component description. Can be used to provide a short summary. Will always appear after the title (if title is set).
```
Comment (bazal4o) [inline en/components/expansion_panel.md]: Which will always appear after ? Place some subject before will

File: en/components/expansion_panel.md [hunk 7]
```diff
@@ -0,0 +1,281 @@
...
+    - `igx-expansion-panel-body` - stores the component body. This part is only visible when the component is expanded.
```
Comment (bazal4o) [inline en/components/expansion_panel.md]: maybe -> stores the components' body. This part of the component is ....

File: en/components/toc.yml [hunk 8]
```diff
@@ -237,6 +237,9 @@
   href: card.md
 - name: Avatar
   href: avatar.md
+- name: Expansion Panel
```
[the remark this answers is not in the document]
Comment (ViktorSlavov) [reply en/components/toc.yml]: I'll add those once the English files are approved so we don't need to update two files at the same time

File: en/components/expansion_panel.md [hunk 9]
```diff
@@ -0,0 +1,586 @@
+---
+title: Expansion Panel
+_description: Provides an easily configurable component with two states - collapsed and expanded.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Collapsible component, IgxExpansionPanel, Ignite UI Expansion Panel
+---
+
+## Expansion Panel
+<p class="highlight">
+The `IgxExpansionPanel` is a lightweigh component which can be rendered in two states - collapsed or expanded. When collapsed, the panel displays a short summary of its content. The collapsed panel can be toggled using mouse click, or keyboard interactions (maybe have link here) to display the body completely, containing any additional necessary content.
```
Comment (bazal4o) [inline en/components/expansion_panel.md]: typo -> lightweight
